transport/server.hh

```diff
@@ -104,6 +104,7 @@ struct cql_query_state {
 struct cql_server_config {
     ::timeout_config timeout_config;
     size_t max_request_size;
+    size_t max_concurrent_requests;
```

Comment:
You use different types for `max_concurrent_requests` (size_t) and for the corresponding cfg parameter `named_value<uint64_t> max_concurrent_cql_requests` .

While on x86_64 it's likely going to be the same `unsigned long long` however it would be safer to use the exactly same integer type (uint64_t) for both.

Reply:
ouch, rookie oversight on my part, will fix in a minute

Reply:
uint64_t seems excessive. Why would we want > 4 billion concurrent requests on the same shard?